### PR TITLE
Feature: Add Survey app specification

### PR DESCRIPTION
**Pull request by jdmedlock**
- Add Survey app specification
- Update the specification template to include a 'Project Board' section
defining the link to the projects Kanban board, currently Trello.

Resolves: N/a
See also: N/a

### CODE REVIEW

File: Projects/Survey-App.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+You can track your progress by cloning this [TBD](url-goes-here)
+
+## Useful links and resources
+
+Libraries for building surveys:
+- [SurveyJS](https://surveyjs.io/Overview/Library/)
+
+Some commercial survey services include:
+- [Survey Monkey](https://www.surveymonkey.com/)
+- [Traversy]()
```
Comment (florinpop17) [inline Projects/Survey-App.md]: You missed the URL here?

File: Projects/Survey-App.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+# Survey
```
Comment (florinpop17) [inline Projects/Survey-App.md]: Maybe add the "App" keyword too: `Survey App`. Make sure to update it in the list in the main README.md

File: Projects/Survey-App.md (hunk 1 above)
Comment (jdmedlock) [reply Projects/Survey-App.md]: Fixed!

File: Projects/Survey-App.md (hunk 2 above)
Comment (jdmedlock) [reply Projects/Survey-App.md]: Fixed

**Pull request merged**